exporter/collector/logs.go

```diff
@@ -418,9 +418,9 @@ func (l logMapper) logToSplitEntries(
 		delete(attrsMap, SourceLocationAttributeKey)
 	}
 
-	// parse TraceSampled boolean from OTel attribute
-	if traceSampled, ok := attrsMap[TraceSampledAttributeKey]; ok {
-		entry.TraceSampled = traceSampled.Bool()
+	// parse TraceSampled boolean from OTel attribute or IsSampled OTLP flag
+	if traceSampled, ok := attrsMap[TraceSampledAttributeKey]; ok || logRecord.Flags().IsSampled() {
+		entry.TraceSampled = (traceSampled.Bool() || logRecord.Flags().IsSampled())
 		delete(attrsMap, TraceSampledAttributeKey)
```

Comment:
Will this fail if the attribute doesn't exist?

Reply:
nope, `delete()` is a no-op if the element doesn't exist. Also added the tests the verify this